### PR TITLE
feat: remove unused postepay logic

### PR DESCRIPTION
**Pull request by pietro-tota**
Here the [related pr](https://github.com/pagopa/pagopa-ecommerce-commons/pull/177) on commons module that remove postepay payment gateway enumeration value

#### List of Changes
Remove unused postepay logic

#### Motivation and Context
Remove unused postepay logic

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -510,6 +510,8 @@
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
                     <scmVersionType>tag</scmVersionType>
                     <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+                    <scmVersionType>branch</scmVersionType>
+                    <scmVersion>CHK-2459-remove-postepay-gateway</scmVersion>
```
Comment (Fernando-Granato) [inline pom.xml:514]: Remember to restore this

**Review by Fernando-Granato — APPROVED** [review]
🚀

File: pom.xml (hunk 1 above)
Comment (pietro-tota) [reply pom.xml]: Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/425/commits/349bd8ee9aef6b2441b5caa9367e35ffe3807b4b

**Review by Fernando-Granato — APPROVED** [review]
🚀

**Review by Fernando-Granato — APPROVED** [review]

**Pull request merged**